frontend/src/routes/Home/Search/searchDefinitions.tsx

```diff
@@ -1367,6 +1360,22 @@ const searchDefinitions: any = {
     },
 }
 
+export function ViolationsCell(compliance: string) {
+    const { t } = useTranslation()
+    if (compliance === 'compliant') {
+        return (
+            <div>
+                <CheckCircleIcon color="var(--pf-global--success-color--100)" /> {t('Without violations')}
+            </div>
+        )
+    }
+    return (
+        <div>
+            <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" /> {t('With violations')}
+        </div>
+    )
+}
+
```

Comment:
I don't think there is any value in factoring this out. The uppercase function name is strange - looks like a component, but not really being used as one.

Reply:
factored out because i wasn't able to get useTranslation hook to work within cell(), but also just refactored slightly so switch statement is also factored out into what im labeling as `GetViolations` now.

Reply:
You could set up `searchDefinitions.tsx` to be a hook just like `useWizardStrings`... that way you also would not need to list all the terms in a big comment. This is much less error-prone, because developers will not know to update the big comment when they add search definitions.

Reply:
reverted functions until revisit changing to hook